experimental/adaptive-dialog/csharp_dotnetcore/02.multi-turn-prompt/Properties/launchSettings.json

```diff
@@ -15,13 +15,13 @@
         "ASPNETCORE_ENVIRONMENT": "Development"
       }
     },
-    "CoreBot": {
+    "MultiTurnPromptBot": {
       "commandName": "Project",
       "launchBrowser": true,
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       },
-      "applicationUrl": "http://localhost:3988/"
+      "applicationUrl": "https://localhost:5001;http://localhost:5000"
```

Comment:
Was the change of port on purpose?

Reply:
Cleaned up launchsettings.json to be consistent. There is no reason to launch on both http and https.